core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/components/dot-category-field-dialog/dot-category-field-dialog.component.html

```diff
@@ -22,15 +22,14 @@
                         (itemChecked)="store.updateSelected($event.selected, $event.item)"
                         (rowClicked)="store.getCategories($event)"
                         [categories]="store.categoryList()"
```

Comment:
There is an error with typing because the input categories is `DotCategoryFieldKeyValueObj[][]` but `.categoryList` return `(DotCategoryFieldKeyValueObj | DotCategoryFieldKeyValueObj[])[]`

Reply:
I cant simulate in Webstorm or VSCode :(

Reply:
We will work in the strict mode in a GD.

Reply:
Ok, in this case please add a comment with `TODO:` for the track this takes in the future.